pkg/helm/helmchart_structure_exists.go

```diff
@@ -130,3 +130,8 @@ func validateChart(chartyamlfile string) (bool, string) {
 
 	return isValid, returnMessage
 }
+
+func GetChartValues(chartYamlFile string) helm.Chartyaml {
```

Comment:
Why not just moving `ChartYamlFromFile` to `pkg` instead of having a one line delegation to `internal`?

Reply:
Moved the function to external helm pkg.

Reply:
@SebastianBezold as we discussed, though it may make sense to move it, it's not a trivial change since current internal unit tests depend on it and it runs into import cycle issue when moved. I suggest to leave it as is keeping it as future candidate for change if we decide on pkgs realignment. What do you think?